### ISSUE TITLE
About Baritone's problem

### ISSUE THREAD

**Issue by KingSonder** [MeteorDevelopment/meteor-client#3874]
### Describe the bug

Automatic mining results in 1.20.1 being able to start but getting stuck when entering the world

### Steps to reproduce

```bash
Open the World
```

### Meteor Version

0.5.4-1869

### Minecraft Version

1.20.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by Big-Iron-Cheems** [comment]
That's definitely not descriptive enough, you need to give us more info about what is happening.

**Comment by Fiz-Victor** [comment]
can you do `#c` and see if this continues to happen

**Comment by KingSonder** [comment]
Yes, every startup will lead to a crash, and the starter always displays Baritone, but later I found that it may be my module matching problem, and thank you for your attention
